### PR TITLE
Fix EDD file uploads

### PR DESCRIPTION
**Pull request by chris48s**
Following on from #447 I missed some permissions we needed to grant to external users

### CODE REVIEW

File: ckanext/unhcr/plugin.py [hunk 1]
```diff
@@ -48,9 +54,11 @@
     'resource_show',
     'resource_update',
     'resource_view_list',
+    'scan_submit',
     'site_read',
     'tag_autocomplete',
     'tag_list',
+    'task_status_show',
```
Comment (chris48s) [inline ckanext/unhcr/plugin.py:61]: I think it is ok to grant `task_status_show` to external users, but I can look at doing it with `ignore_auth: True` instead if you want

File: ckanext/unhcr/plugin.py [hunk 2]
```diff
@@ -29,6 +29,12 @@
 
 
 ALLOWED_ACTIONS = [
+    'cloudstorage_initiate_multipart',
+    'cloudstorage_upload_multipart',
+    'cloudstorage_finish_multipart',
+    'cloudstorage_abort_multipart',
+    'cloudstorage_check_multipart',
+    'cloudstorage_clean_multipart',
```
Comment (chris48s) [inline ckanext/unhcr/plugin.py:37]: refs https://github.com/okfn/ckanext-cloudstorage/pull/6

File: ckanext/unhcr/plugin.py (hunk 1 above)
Comment (amercader) [reply ckanext/unhcr/plugin.py:61]: I think it's safe to allow this

**Review by amercader — APPROVED** [review]
Looks good!

**Pull request merged**